### DISCUSSION TITLE
请问能不能切换菜单的时候，不自动修改浏览器标签的标题？

### DISCUSSION THREAD

**Discussion by ShadowJobs** [ant-design/ant-design-pro#10196] [Q&A] [answered]
这个设计太坑了，老板现在让我把标题固定住，我都不知道怎么跟他解释。他问为什么别的网站都能固定住。
我尝试写在document.ejs，defaultsettings的title，都不管用
我尝试在app.tsx里加title，结果，加倒是加上 ，但是加在了原来的菜单内容的后面T_T

**Comment by neveler** [comment, marked as answer]
文档里好像有 https://pro.ant.design/zh-CN/docs/title-landing#%E6%A0%87%E9%A2%98%E5%92%8C-logo

![image](https://user-images.githubusercontent.com/55753029/188259738-88506235-99ec-4c99-9603-9a75811a3035.png)

**Reply by ShadowJobs** [reply]
我试过直接写title，没有生效。
根据您的回复，我又看了一遍。关键是这一句：pageTitleRender=false
感谢回复。

**Reply by jkiss** [reply]
今天测试还给我提了 “返回登录页面，title没有变化的bug” 🐶

**Reply by neveler** [reply]
> 今天测试还给我提了 “返回登录页面，title没有变化的bug” 🐶

不知道我理解的对不对：

登录界面路由一般都是 layout: false , 所以不继承 title , 所以根据文档建议可以使用 document.title 实现`document.title = '登录';`。

**Reply by jkiss** [reply]
我之前5.0版本起的项目就可以改变，6.0 beta1 起的项目就不行了😂，不过最后也是用 document.title 手动设置了
